### ISSUE TITLE
Certificate revocation

### ISSUE THREAD

**Issue by enricomiletto** [eu-digital-green-certificates/dgc-overview#45]
## Your Issue

I would like to ask wether there's some kind of certificate revocation list that's exchanged in the EU gateway alongside with the list of Public Keys.

And if it's not the case (which what I've read in a discussion in the repo of the italian verifier app) I would like to ask the reasoning behind this, as it seems like a pretty important feature.

Are there privacy concerns maybe?

**Comment by postulino** [comment]
There should not be a privacy concern, issue or revoke a document is the same. Revoke could give an important figure for health data in EEA and give a better figure on campaign effectivness.

**Comment by enricomiletto** [comment]
> I would like to ask wether there's some kind of certificate revocation list

Just to clarify, I'm referring to the single Digital Green Certificates (DGCs) here, not the DSCs

**Comment by SchulzeStTSI** [comment]
The revocation of a single DCC certificate is still a discussed topic. Technically are there proposals on the table, but there are a lot of concerns. For instance how you identify the right DCC for revocation(especially with decentralized issuers), how to exchange the lists and how effective a revocation can be. However you identify a certificate and/or the person behind it, you can revoke it, but it needs some hours/days to share all this information across all verifier devices for offline verification. A Test DCC revocation with 48 Hours validity is then not really reachable by a revocation list. For vaccination and recovery would be the revocation of the issuer much more effective than for a single person. So there are a lot of controverse discussion points about single dcc revocation.

**Comment by enricomiletto** [comment]
@SchulzeStTSI thanks for taking the time to answer.

> how you identify the right DCC for revocation

Are the unique identifiers not unique for each DCC?
I actually thought that revocation was the main reason for the existence of the UIs, so could you maybe elaborate on why this is still a source of discussion?

**Comment by enricomiletto** [comment]
@SchulzeStTSI any updates?

**Comment by daniel-eder** [comment]
@enricomiletto I am not sure if there is any new status on that discussion (I assume @SchulzeStTSI will post here if there are news), but in the interest of transparency it should be mentioned that this is not a decision made on developer side, but rather by the European Union and their advisory bodies. 

As a further explanation, certificate revocation is not purely a technical issue, legal (and political) aspects play into it, such as e.g. data privacy protection, which, depending on the exact context, can directly affect the ability to identify the correct DCCs for revocation. 

In short, there's a lot of aspects to this topic, and a lot of different concerns that need to be coordinated on a political level before it can move forward.

**Comment by enricomiletto** [comment]
@daniel-eder thanks a lot for the explanation

**Comment by bhavin-qryptal** [comment]
@daniel-eder , @SchulzeStTSI  Are there any new development on this front? Thank you very much for the enlightenment.
